hypothesis-python/src/hypothesis/extra/numpy.py

```diff
@@ -96,6 +97,34 @@ def _try_import(mod_name: str, attr_name: str) -> Any:
 NP_FIXED_UNICODE = tuple(int(x) for x in np.__version__.split(".")[:2]) >= (1, 19)
 
 
+def _is_comparable(x: Any) -> bool:
+    """Returns True if the input can be compared."""
+    try:
+        return x == x or x != x
+    except Exception:
+        return False
+
+
+def _has_ssize_t_length_or_no_length(x: Any) -> bool:
+    """Returns True if the input has a __len__ method that returns an integer that fits into ssize_t.
+
+    This is used to so that we convert large object arrays to strings we don't get OverflowErrors when printing.
+    E.g. Pandas will test len() of an entry in order to print it. This will fail if the length is too large.
+    """
+    try:
+        len(x)
+    except OverflowError:
+        return False
```

Comment:
It looks like the coverage tests are reporting that this line was not executed.

Reply:
Thanks. Out of interest, how do you find that out?